persistence-test/src/main/java/io/stargate/it/storage/ExternalStorage.java

```diff
@@ -293,7 +293,7 @@ public void stop() {
           }
         } catch (Exception e) {
           // This should not affect test result validity, hence logging as WARN
-          LOG.warn("Exception during CCM cluster shutdown: {}", e, e);
+          LOG.warn("Exception during CCM cluster shutdown: " + e.getMessage(), e);
```

Comment:
Would have failed to do what is expected: overload was taking both `e`s as substitutions (but intent was to include stack trace).